tools/check_make_proto.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+source build.env
+
+first_output=$(git status --porcelain)
```

Comment:
Wouldn't an initial `git status` on a fresh checkout always be empty?

Reply:
Yes, technically the first git status should be empty, however, I wanted to make sure that the only possible reason for a drift comes from make proto, no matter how or when this script is being run. So I assumed the script could be run in any situation/environment, and not necessarily on a clean git state :)